website/src/controllers/api-v3/auth.js

```diff
@@ -210,7 +209,7 @@ api.loginLocal = {
     let user = await User.findOne(login, {auth: 1, apiToken: 1}).exec();
 
     // TODO place back long error message return res.json(401, {err:"Uh-oh - your username or password is incorrect.\n- Make sure your username or email is typed correctly.\n- You may have signed up with Facebook, not email. Double-check by trying Facebook login.\n- If you forgot your password, click \"Forgot Password\"."});
```

Comment:
@piousbox can you add this message as a localized string in place of the shorter message at line 215?